Use reST links for downloads

cjmayo opened the pull request:
Use the same decoration with dcclink class but replace the raw HTML (except for the one using JavaScript) and document in Standards.

Draft because it goes after the changes in #136. This PR is only the last commit.

FrightRisk commented:
That seems like a good idea. I appreciate you bringing some knowledge about Sphinx and markup to tidying things up.

FrightRisk commented:
Looks like you are ready to do what you need to to move this from a draft to a PR. Everything else is merged.

cjmayo committed:
Use reST links for downloads

Use rst-class to apply the dcclink class to the paragraph containing the
anchor.

cjmayo commented:
Ready to go. Rebased on the updated sphinx branch.
